features/nfc/nfc/NFCController.h

```diff
@@ -1,6 +1,8 @@
 /* mbed Microcontroller Library
  * Copyright (c) 2018 ARM Limited
  *
+ * @file NFCController.h
```

Comment:
That is not going to do anything as it is not part of a Doxygen comment. The same is true for other headers modified.

Reply:
@pan- \file or @file [<Name>] should be standard doxygen specifier. Do you mean that there is some option that we need to turn on while generating doxygen in the doxy options file  ?

Reply:
Interesting, I have seen many examples where file description is added alongwith license text. I can try to add a separate doxy section for @file.

Reply:
@hasnainvirk I'm not saying `@file` isn't a valid directive. I'm just pointing out it should be part of a Doxygen comment. A doxygen comment starts with `/**`, `/*!`, `///` or `//!`. In this case the licence documentation block starts with `/*`

Reply:
Adjusted the license block instead.

Reply:
@pan- Yeah, figured that out while you were writing your message :)